internal/certmanager/cm_controller.go

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright 2020 The cert-manager Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package certmanager provides a controller for creating and managing
+// certificates for VS resources.
+package certmanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/workqueue"
+
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
Any reason not to update these to the new cert-manager location now?

Reply:
The only reason was that 1.8.0 (the first release to use the new repository location) was not released yet. I see it was released 2 days ago, I'll update to the latest version and update the module paths accordingly 👍🏼